src/action-types-array.js

```diff
@@ -0,0 +1,32 @@
+import { TypeError, UnicityError } from './errors.js';
+// ------------------------------------ //
+const isString = arg => typeof arg === 'string' || arg instanceof String;
+// ------------------------------------ //
+const StringOrThrow = element => {
+  if (!isString(element)) throw new Error(TypeError);
+};
+// ------------------------------------ //
+const RaiseErrorIfNotUnique = array => {
+  if (array.length !== new Set(array).size) {
```

Comment:
No constructor support  for `Set` like this in <=IE11, so this will transpile I assume.  Can you confirm, and into what kind of code?

Reply:
I've updated this with a single loop handling. 
Set seem to be untouched by Babel.

"Babel can't "translate" Map and Set, because they aren't language features (although they are described in the ES spec). They're classes that exist in the global scope.

https://stackoverflow.com/questions/40388530/babel-es2015-presets-doesnt-translate-map-and-set-to-es5